versioned_docs/version-8.5/components/modeler/web-modeler/launch-web-modeler.md

```diff
@@ -14,7 +14,7 @@ To launch Web Modeler, follow the steps below:
 2. Select **Create new project** to create a new project and store diagrams.
    ![web modeler empty home](img/web-modeler-new-user-home.png)
 3. Name your diagram. You can go back and change the name any time by clicking on the project name and **Edit name**.
-4. Select **Browse blueprints** to view blueprints for various use cases as a starting point for your first diagram. Open these blueprints by selecting **Use Blueprint**. Alternatively, click **Create new > BPMN diagram** to create a blank BPMN diagrams.
+4. Select **Browse blueprints** to view blueprints for various use cases as a starting point for your first diagram. Open these blueprints by selecting **Use Blueprint**. Alternatively, click **Create new > BPMN diagram** to create a blank BPMN diagram.
```

Comment:
Is this inconsistently cased in the product? 

Browse blueprints vs. Use Blueprints?

Reply:
It is, yes. I can likely launch a PR in the product to adjust!

Reply:
The confusion is from the text, not the product. The user clicks "Browse blueprints" to open the blueprints dialog. Once the user finds a blueprint they want to use, they need to click on "Use blueprint," which will download the blueprint into the project (and open it in the diagram screen).

Reply:
Here is what I currently see in the product:

<img width="1120" alt="Screenshot 2024-09-26 at 9 19 07 AM" src="https://github.com/user-attachments/assets/0661d6a8-561f-413e-a2ea-b14707e8db3f">
<img width="907" alt="Screenshot 2024-09-26 at 9 19 20 AM" src="https://github.com/user-attachments/assets/bf98abae-a10d-4541-b8c1-2fa1b090999a">

Reply:
Are you saying that you would prefer to see the text rewritten as:

Select **Browse blueprints** to open the blueprints dialog and view blueprints for various use cases as a starting point for your first diagram. Open these blueprints by selecting **Use Blueprint**, which will download the blueprint into the project and open it in the diagram screen. Alternatively, click **Create new > BPMN diagram** to create a blank BPMN diagram.

If so, I'll open a separate issue to revisit this.

Reply:
Yes excatly. Thank you :)